### DISCUSSION TITLE
I've been waiting for github copilot for a long time and I still haven't got access from github

### DISCUSSION THREAD

**Discussion by NLTD2010** [community/community#10193] [Copilot Conversations] [answered]
pls rep me

**Comment by NLTD2010** [comment]
ps help me

**Comment by NLTD2010** [comment]
i have code exam tomorow

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see [#9342 (comment)](https://github.com/github/feedback/discussions/9342#discussioncomment-1902060)
